### PR TITLE
Fix/analytics

### PR DESCRIPTION
**Pull request by blockdylanb**
## Description (optional)
Add a concise explanation of the changes.

## Testing Steps (optional)
Detail the steps required for the reviewer(s) to verify and test these changes.

### CODE REVIEW

File: packages/blockchain-wallet-v4-frontend/src/middleware/analyticsMiddleware/index.ts [hunk 1]
```diff
@@ -64,6 +64,17 @@ const analyticsMiddleware = () => (store) => (next) => (action) => {
 
         const pageName: PageName = action.payload.location.pathname
 
+        try {
+          // @ts-ignore
+          if (window && window.gtag) {
+            // @ts-ignore
+            window.gtag('set', 'page_path', pageName)
+            // @ts-ignore
```
Comment (schnogz) [inline packages/blockchain-wallet-v4-frontend/src/middleware/analyticsMiddleware/index.ts:72]: add `gtag` to `src/typings/window.d.ts` so you don't have to use `@ts-ignore`'s

**Review by schnogz — CHANGES_REQUESTED** [review]

File: packages/blockchain-wallet-v4-frontend/src/index.html [hunk 2]
```diff
@@ -34,6 +34,15 @@
...
       window.nonce = '**CSP_NONCE**'
     </script>
+    <script nonce="**CSP_NONCE**">
+      window.dataLayer = window.dataLayer || [];
+      function gtag(){dataLayer.push(arguments);}
+      gtag('js', new Date());
+      gtag('config', 'UA-52108117-5');
+      gtag('config', 'UA-52108117-5', {
+        send_page_view: false
+      });
```
Comment (milan-bc) [inline packages/blockchain-wallet-v4-frontend/src/index.html:44]: do we need both lines of one is enough? for config

File: packages/blockchain-wallet-v4-frontend/src/middleware/analyticsMiddleware/index.ts (hunk 1 above)
Comment (milan-bc) [reply packages/blockchain-wallet-v4-frontend/src/middleware/analyticsMiddleware/index.ts]: there is dedicated typing https://github.com/DefinitelyTyped/DefinitelyTyped/tree/4d4da2106dcd54af4a0ddeb5e0360f322fd8a5bf/types/gtag.js

**Review by milan-bc — APPROVED** [review]

File: packages/blockchain-wallet-v4-frontend/src/middleware/analyticsMiddleware/index.ts (hunk 1 above)
Comment (blockdylanb) [reply packages/blockchain-wallet-v4-frontend/src/middleware/analyticsMiddleware/index.ts]: Even from beyond the grave @schnogz still got our backs. <3

**Pull request merged**